src/views/home/home.vue

```diff
@@ -66,6 +74,14 @@ export default class Home extends Vue {
     this.locationPermissionLink = this.getLocationPermissionLink();
   }
 
+  public created() {
```

Comment:
I was curious about this method registering the `setInterval()`. Should there be a check prior to registration to make sure we have acquired permission to get location data?

so something to the effect of

```
public created() {
 if (this.locationPermissionActivated) {
   this.pollData();
 } else {
  //Some action to encourage them to activate location services
 }
}

```

Reply:
Okay, I updated this to call the `pollData` method from the beforeMount call where we are setting that value.

Let me know what you think. 👍

Reply:
@prowe214 For your `AlertBanner` recommendation, check out the code I added to the AppComponent to allow us to trigger an alert from any child component.